src/node.cc

```diff
@@ -4556,7 +4557,9 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   {
     SealHandleScope seal(isolate);
     bool more;
+    PERFORMANCE_MARK(&env, LOOP_START);
     do {
+      PERFORMANCE_MARK(&env, LOOP_FRAME);
```

Comment:
basically, with the Platform PR landing, it may not make sense to keep `LOOP_FRAME`. We have the `LOOP_PREPARE`, `LOOP_CHECK` and `LOOP_IDLE` milestones that can be used. The key issue, however, would be calculating duration.